docs/getting_started/ai-resources.md

```diff
@@ -0,0 +1,73 @@
+# AI Trainings and Resources Guide
+
+## CU Boulder & CURC Resources
+
+### CU Boulder: AI Efforts on Campus
+- Overview of AI initiatives and support across campus. [https://www.colorado.edu/information-technology/ai-cu-boulder/ai-efforts-campus](https://www.colorado.edu/information-technology/ai-cu-boulder/ai-efforts-campus)
+### RMACC-CU AI Workshop
+- Workshops held in partnership with the Rocky Mountain Advanced Computing Consortium. [https://www.colorado.edu/rc/cu-rmacc-ai-workshop](https://www.colorado.edu/rc/cu-rmacc-ai-workshop)
+### CURC AI/ML Trainings
+- CURC offers trainings each semester to help researchers deploy and optimize AI/ML workflows.
+    * [Setting up LLMs on CURC Resources (an RC Short Course)](https://github.com/ResearchComputing/setting_up_llms_on_curc_resources_rc_short_course)
+    * [Applied Containerization for Machine Learning in HPC (an RC Short Course)](https://colorado.libcal.com/calendar/events/contain)
+- For a complete list of current semester offerings, see [CURC Trainings Page](../getting_started/current-sem-trainings.md)
+### CURC AI/ML Software Stack
+- The following AI/ML software are available and actively supported on CURC systems:
+    * PyTorch (1.13.0): Popular deep learning framework supporting GPU acceleration.
```

Comment:
This is an insanely outdated version of PyTorch and is only available for our AMD GPUs. Also, I do not want to be in the business of maintaining PyTorch. I suggest we remove this. I will eventually push for removing PyTorch in favor of an RTD page. 

I would then add the `ollama` and `hf-transformers` modules I added.